medusa/show/recommendations/recommended.py

```diff
@@ -42,6 +44,7 @@
 log.logger.addHandler(logging.NullHandler())
 
 session = MedusaSession()
+VALID_INDEXERS = {indexerConfig[indexer]['identifier']: indexer for indexer in indexerConfig}
```

Comment:
I feel like this should be defined in indexer config and imported. Why `VALID_INDEXERS`? What makes them valid?

Reply:
Indexers we use to add shows. Oppositie to imdb and trakt.

Reply:
I would keep this as `INDEXERS` and eventually the ones that only give info trakt, imdb as `INFO_INDEXERS`? Or even not call them indexers at all.

Reply:
`TV_SOURCE` ?

Reply:
I'll keep it as INDEXERS, else it's EXTERNALS

Reply:
I moved and renamed it. Put it in indexer.utils.